### DISCUSSION TITLE
Updating Site for 2023 season

### DISCUSSION THREAD

**Discussion by Jimbobmcghee** [nmelhado/league-page#259] [Help!]
I would like to apologize in advance for what is likely a very dumb question. 

I had a functional site last year which my league loved. I'm unable to merge updates due to conflicts (my league info in leagueinfo.js). I tried to create a new league page with the updates in place, however I am still showing 2022 rosters and results. I have very little coding experience, so any tips that anyone could offer to update the page would be great!

Thanks for this!

**Comment by nmelhado** [comment]
Every season, Sleeper assigns new league IDs to every league. You need to update your leagueInfo with the new league ID in order to get this current season.

If that doesn't work, let me know!

**Reply by Jimbobmcghee** [reply]
I can't believe I didn't notice that... I just assumed the ID would stay the same so I tried to fix just about everything else other than the ID. 

It worked, thanks so much! Love the site!

**Reply by nmelhado** [reply]
No worries, it's a bizarre approach to have the league ID change every year. Definitely not intuitive.
